cfpb/cfgov-refresh
# Upgrade Capital Framework components and use their brand colors instead of cf-theme-overrides

Use the latest Capital Framework components so that we no longer have to maintain brand colors in this repository.

## Removals

- Deletes `cf-theme-overrides.less` and `brand-palette.less` from the project and removes them from `main.less`

## Changes

- CFPB-branded color variables are now defined directly in the Capital Framework components that are upgraded in `package.json`
- Color updates for CF components should be managed upstream in Capital Framework's [`cf-brand-colors.less`](https://github.com/cfpb/capital-framework/blob/master/src/cf-core/src/cf-brand-colors.less) file

## Testing

1. You can test this locally, but pretty much all the colors will remain unchanged on the front end, because most variables are duplicated in this repo's atomic components anyway.
1. check out this branch
1. run `./frontend.sh`
1. `./runserver.sh`and `gulp watch` in another window
1. confirm styles are loading as expected

## Screenshots


## Notes

-

## Todos

-

## Checklist
